setup.py

```diff
@@ -39,7 +39,7 @@ def load_source(modname, filename):
         "Programming Language :: Python :: 3.12",
         "Programming Language :: Python :: 3.13",
     ],
-    python_requires=">=3.9",
+    python_requires=">=3.10",
```

Comment:
3.9 is still live, we shouldn't drop it yet.

Reply:
my bad, i misread the comment in the other thread. i'll work on the namespace issue then!